src/Lounge.Web/Pages/Stats.cshtml

```diff
@@ -93,8 +93,8 @@
 <h1 class="m-4 text-center">@Localizer["Activity Data"]</h1>
 
 <div class="row m-2">
-    <canvas id="statMogiFormatChartBody" class="col-lg-6 col-md-6 col-sm-6 col-xs-6"></canvas>
-    <div class="table-responsive my-auto col-lg-6 col-md-6 col-sm-6 col-xs-6" >
+    <canvas id="statMogiFormatChartBody" class="col-lg-6 col-md-12 col-sm-12 col-xs-12"></canvas>
```

Comment:
fyi, I believe you don't need to specify every size, if you do col-md-12 it will use 12 for sm and xs as well, basically it takes whatever is the next highest size.